plugins/rapid7_insightvm_cloud/help.md

```diff
@@ -42,6 +42,147 @@ Example input:
 
 ### Actions
 
+#### Vulnerability Search
+
+This action is used to search for vulnerabilities using filtered vulnerability search.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|asset_criteria|string|None|False|Filters to apply to the asset search such as IPv4 or IPv6 addresses and hostnames|None|asset.ipv4 = 2001:db8:1:1:1:1:1:1 || asset.name STARTS WITH 'example'|", "|vuln_criteria|string|None|False|Vulnerability criteria to filter by|None|vulnerability.categories IN ['example']|
+|size|integer|200|False|The number of vulnerabilities to retrieve. If blank then will default to 200 vulnerabilities returned, the maximum limit is 500 vulnerabilities|None|100|
+|sort_criteria|object|None|False|JSON object for sorting by criteria. Multiple criteria can be specified with an order of 'asc' (ascending) or 'desc' (descending)|None|{'risk-score': 'asc', 'criticality-tag': 'desc'}|
+|vuln_criteria|string|None|False|Vulnerability criteria to filter by|None|https://example.com IN ['example']|
+
+Example input:
+
+```
+{
+  "asset_criteria": "asset.ipv4 = 2001:db8:1:1:1:1:1:1 || asset.name STARTS WITH 'example'",
+  "size": 100,
+  "sort_criteria": "{'risk-score': 'asc', 'criticality-tag': 'desc'}",
```

Comment:
Does it has to stay here as string? Or should that be showed as object? 😄

Reply:
It's has to stay as string, yes.